### PR TITLE
[DOC-243] [skip-ci] Sql ingest pipeline template docs

### PR DESCRIPTION
**Pull request by nadegepepin**

### CODE REVIEW

File: doc/docs/master/how-tos/basic-data-operations/sql-ingest.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+Our **SQL ingest** tool provides a seamless connection between databases and Pachyderm,  allowing you to import data from a SQL database into Pachyderm-powered pipelines. By bringing data-driven pipelines, versioning & lineage to structured data, we are allowing Data Science teams to easily combine structured and unstructured data, thus improving the performance of prediction models.
+
+Specifically, we help you connect to a remote database of your choice and pull the result of a given query at regular intervals in the form of a CSV or a JSON file.  
+
+## Use SQL Ingest
+Pachyderm's SQL Ingest uses [pipeline templates](../../pipeline-operations/pipeline-templates) populated with the following parameters to automatically create the pipelines that access, query, and materialize the results in the form of CSV or JSON files.
+
+Pass in the following parameters and get your results committed to an output repo, ready for the following downstream pipeline:
+```shell
+pachctl update pipeline --jsonnet ./src/templates/sql_ingest_cron.jsonnet
```
Comment (brendoncarroll) [inline doc/docs/master/how-tos/basic-data-operations/sql-ingest.md:15]: This should be the url to master when we merge.  Maybe we can change it to the release commit url at some point

File: doc/docs/master/how-tos/basic-data-operations/sql-ingest.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+| `secretName`  | The kubernetes secret name that contains the [password to the database](#database-secret).|
+
+!!! Note
+    `pachctl update pipeline` will create pipelines if none exist or update otherwise.
+
+
+When the pipeline template command is run, the database will be queried on a schedule defined in your `cronSpec` parameter and a result file committed to the output repo named after `name`.
+
+### Database Secret
+Before you create your SQL Ingest pipelines, make sure to create a [generic secret](../advanced-data-operations/secrets/#create-a-secret) containing your database password in the field `PACHYDERM_SQL_PASSWORD`.
```
Comment (brendoncarroll) [inline doc/docs/master/how-tos/basic-data-operations/sql-ingest.md:42]: 👍

**Review by brendoncarroll — APPROVED** [review]
I think we should mention that downstream pipelines should be prepared to consume multiple files as a best practice.  Otherwise the 0000 seems sort of obscure, and not adequately motivated.

File: doc/docs/master/how-tos/basic-data-operations/sql-ingest.md (hunk 1 above)
Comment (nadegepepin) [reply doc/docs/master/how-tos/basic-data-operations/sql-ingest.md]: Set `https://raw.githubusercontent.com/pachyderm/pachyderm/{{ config.search_index_version }}/src/templates/sql_ingest_cron.jsonnet ` instead where {{ config.search_index_version }} comes from the config file.
i.e: `master` now, `2.1.x` once released.

**Pull request merged**